setup.py

```diff
@@ -38,7 +38,15 @@
 License :: OSI Approved
 Programming Language :: C
 Programming Language :: Python
+Programming Language :: Python :: 2
+Programming Language :: Python :: 2.6
+Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
+Programming Language :: Python :: 3.2
+Programming Language :: Python :: 3.3
```

Comment:
According to [1.11.0-notes.rst](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/doc/release/1.11.0-notes.rst), [1.10.0-notes.rst](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/doc/release/1.10.0-notes.rst), and [setup.py](https://github.com/numpy/numpy/blob/9cd868a00cc488478ac4f07ee0b5452a38cb3cc0/setup.py#L25) Python 3.2 is supported.